mqtt.kafka.reflect/README.md

```diff
@@ -125,6 +126,26 @@ $ kcat -C -b localhost:9092 -t mqtt_messages -J -u | jq .
 % Reached end of topic mqtt_messages [0] at offset 1
 ```
 
+Verify retained messages
+```bash
+$ mosquitto_pub -V '5' -t 'zilla' -m 'Retained message' -d --retain
+Client null sending CONNECT
+Client 42adb530-b483-4c73-9682-6fcc370ba871 received CONNACK (0)
+Client 42adb530-b483-4c73-9682-6fcc370ba871 sending PUBLISH (d0, q0, r1, m1, 'zilla', ... (16 bytes))
+Client 42adb530-b483-4c73-9682-6fcc370ba871 sending DISCONNECT
```

Comment:
Perhaps we could publish 2 messages -- first one retained, second one not retained -- then show that subscribe receives only the earlier retained message?

This proves that we have `mqtt` semantics of keep only retained messages, rather than `kafka` semantics where either every message (cleanup=delete) or the last message (cleanup=compact) is effectively "retained".

It would also be good to show that if another retained message is published, only the most recent retained message is delivered to the mqtt subscriber.